drivers/scheduler/k8s/specs/debug/debug.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       priorityClassName: system-cluster-critical
       containers:
       - name: debug
-        image: ubuntu:latest
+        image: harbor.pwx.dev.purestorage.com:443/px-image/ubuntu:latest
```

Comment:
Not sure if we want to commit harbor pulls since its actually only accessible from certain network.

Reply:
this might fail existig runs

Reply:
can u run this job with utr private image
https://jenkins.portworx.dev/job/Torpedo/job/tp-dev-test/

Reply:
removed the charts. will work on running the job